jobs/client-regeneration/client_regeneration/main.py

```diff
@@ -302,6 +302,109 @@ def update_churn_pool(client, seed, date):
 #     return table_name
 #
 
+def write_baseline_clients_daily(client, seed, start_date):
+  table_name = """mozdata.analysis.regen_sim_replaced_baseline_clients_daily_{}""".format(str(seed))
+  q = f"""
+  CREATE OR REPLACE TABLE {table_name}
+  AS
+  WITH base AS (
+      SELECT
+        COALESCE(r.replacement_id, c.client_id) AS client_id,
+        COALESCE(r.first_seen_date, c.first_seen_date) AS first_seen_date,
+        c.client_id AS original_client_id,
+        c.first_seen_date AS original_first_seen_date,
+        c.submission_date,
+        c.country,
+        c.device_model,
+        regened_last_date,
+
```

Comment:
Do we need a `sample_id` as well? I believe that was used for train/test split in the notebook